packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,137 @@ section: components
 related: ['Alert', 'Alert group']
 ---
 
-An **alert** is used to notify the user about a change in status or other event without blocking other actions in an interface. These guidelines talk about how to properly use different types of alerts in your designs.
+An **alert** is a notification that provides brief information to the user without blocking their workflow.
+
+There are two main types of alerts:  
+
+- **Inline alerts**: Inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
+
+![inline notification placement](./img/inline-notification.png)
+
+- **Toast alerts**: Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlys content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
+
+![toast notification placement](./img/toast-notification.png)
 
 ## Elements
-### Alert
-Two styles can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Toast alerts should always be dismissable, whereas inline alerts can be both dismissable and non-dismissable. Other elements are consistent between alert types.
+Toast alerts are always dismissible, and inline alerts can be both dismissable and non-dismissible. Other elements are consistent between both alert types.
 
 <img src="./img/alert-types.png" alt="Alert types" />
```

Comment:
This is small and I'm sorry I didn't notice this earlier, but do ming changing the font of the headers in the image to Red Hat Text instead of overpass?